packages/botonic-react/src/webview-app.tsx

```diff
@@ -8,18 +8,59 @@ import { BrowserRouter, Route } from 'react-router-dom'
 import { CloseWebviewOptions, WebviewRequestContext } from './contexts'
 
 class App extends React.Component {
+  private url: URL
   constructor(props) {
     super(props)
-    const url = new URL(window.location.href)
-    const params = Array.from(url.searchParams.entries())
+    this.url = new URL(window.location.href)
+    this.state = {
+      session: null,
+      params: {},
+    }
+  }
+
+  async componentDidMount() {
+    try {
+      const botId = this.url.searchParams.get('bot_id')
+      const chatId = this.url.searchParams.get('chat_id')
+      if (botId && chatId) {
+        this.getBotSessionContextFromExternalApi(botId, chatId)
+      } else {
+        this.getBotSessionContextFromUrl()
+      }
+    } catch (error) {
+      this.getBotSessionContextFromUrl()
+    }
+  }
+
+  async getBotSessionContextFromExternalApi(botId: string, chatId: string) {
+    // const baseUrl = session._hubtype_api || 'https://api.hubtype.com'
+    const baseUrl = 'https://api.hubtype.com' // TODO: How to retrieve here the api url to request?
```

Comment:
We have to think about how to solve this, otherwise in non-production environments it won't work

Reply:
we have added a `hubtype_api_url` param to have this